packages/manager/modules/pci/src/projects/project/serving/namespace/infos/infos.html

```diff
@@ -0,0 +1,78 @@
+<cui-message-container data-messages="$ctrl.messages"></cui-message-container>
+<div class="row d-lg-flex">
+    <div class="col-md-4">
+        <oui-tile data-heading="{{ ::'pci_projects_project_serving_namespace_infos_namespace_information_management' | translate }}">
+            <oui-tile-button
+                data-on-click="$ctrl.addModel()">
+                <span data-translate="pci_projects_project_serving_namespace_infos_namespace_information_management_deploy_model"></span>
+            </oui-tile-button>
+            <oui-tile-button
+                data-on-click="$ctrl.listModels()">
+                <span data-translate="pci_projects_project_serving_namespace_infos_namespace_information_management_list_model"></span>
+            </oui-tile-button>
+            <oui-tile-button
+                data-ng-if="!$ctrl.registry.custom"
+                data-on-click="$ctrl.attachRegistry()">
+                <span data-translate="pci_projects_project_serving_namespace_infos_namespace_information_management_registry_attach"></span>
+            </oui-tile-button>
+            <oui-tile-button
+                data-ng-if="$ctrl.registry.custom"
+                data-on-click="$ctrl.detachRegistry()">
+                <span data-translate="pci_projects_project_serving_namespace_infos_namespace_information_management_registry_detach"></span>
+            </oui-tile-button>
+            <oui-tile-button
+                data-on-click="$ctrl.deleteNamespace()">
+                <span data-translate="pci_projects_project_serving_delete_label"></span>
+            </oui-tile-button>
+        </oui-tile>
+    </div>
+    <div class="col-md-4">
+        <oui-tile class="h-100" data-heading="{{:: 'pci_projects_project_serving_namespace_infos_namespace_information' | translate }}">
+            <oui-tile-definition data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_description' | translate }}">
+                <oui-tile-description data-ng-bind="$ctrl.namespace.description"></oui-tile-description>
+            </oui-tile-definition>
+            <oui-tile-definition data-term="{{:: 'pci_projects_project_serving_namespace_infos_namespace_region' | translate }}">
+                <oui-tile-description data-ng-bind="$ctrl.namespace.region"></oui-tile-description>
+            </oui-tile-definition>
+            <div class="oui-tile__item">
+                <dl class="oui-tile__definition">
+                    <dt class="oui-tile__term">
+                        <span data-translate="pci_projects_project_serving_namespace_infos_namespace_container"></span>
+                    </dt>
+                    <dd class="oui-tile__description d-flex align-items-left" data-ng-click="$ctrl.goToContainer()">
+                        {{ $ctrl.namespace.container }}
```

Comment:
```suggestion
                        <span data-ng-bind="$ctrl.namespace.container"></span>
```
because it is better for performances 🙂